### PR TITLE
[feature] Created the proposal for a Roman2Dec-App idea.

### PR DESCRIPTION
**Pull request by vbarzana**
Hi @florinpop17,
Please see this app request, not sure if would fit the standards, but just came into my mind. Most of the content I copied from the wikipedia but I checked that is correct.
The idea is that the user is able to create a Roman to Decimal converter.
Regards,
Victor

### CODE REVIEW

**Comment by florinpop17** [comment]
Hey @vbarzana. Thank you for your submission!
We've checked it and we believe that there is too much explanation going on in the project file. Most of the people should know what a Roman number is.

Could you remove the text and maybe just add a wikipedia link for those who want to find out more about the Roman numbers. Focus on the User stories and Bonus features (which look pretty good, so you can keep them).

Let us know when you make the changes!

**Comment by vbarzana** [comment]
@florinpop17 Please take a look now, should be fine, although I still left a bit of introduction with the wiki link to read more.
Thanks,
Victor

File: Projects/Roman2Dec-App.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ 	
+ 	I - 1
+ 	V - 5
+ 	X - 10
+ 	L - 50
+ 	C - 100
+ 	D - 500
+ 	M - 1000
+
+Please take a look in Wikipedia [for more details about Roman Numbers](https://en.wikipedia.org/wiki/Roman_numerals)
```
Comment (florinpop17) [inline Projects/Roman2Dec-App.md]: I would remove this link too as it is in the Useful Links section down below!

Comment (vbarzana) [reply Projects/Roman2Dec-App.md]: Done!

**Comment by florinpop17** [comment]
Thank you for your work @vbarzana !! We'll be waiting for your next idea! 😄

**Pull request merged**